### ISSUE TITLE
Subscriptions with API Gateway v2

### ISSUE THREAD

**Issue by hakimio** [graphile/postgraphile-lambda-example#12]
It would be nice if this project would include graphql subscriptions using API Gateway v2 like shown in [this demo project](https://github.com/michalkvasnicak/aws-lambda-graphql).

**Comment by benjie** [comment]
From a quick scan of the README it looks like that works by having DynamoDB directly trigger the lambda instance. With PostgreSQL we have LISTEN/NOTIFY and logical decoding, but they can't trigger lambda - we need a persistent server to monitor the LISTEN/NOTIFY / logical decoding, and then trigger lambda, which kind of defeats the purpose since we may as well just run PostGraphile on that dedicated server. If you know otherwise, please share.

TL;DR: I don't think we can achieve this without a separate dedicated server.

**Comment by hakimio** [comment]
What do you think about [this approach using wal2json](https://aws.amazon.com/blogs/database/stream-changes-from-amazon-rds-for-postgresql-using-amazon-kinesis-data-streams-and-aws-lambda/)?
